portality/api/v1/discovery.py

```diff
@@ -140,16 +184,35 @@ def search_journals(cls, q, page, page_size):
             raise DiscoveryException("There was an error executing your query")
 
         obs = [models.Journal(**raw) for raw in esprit.raw.unpack_json_result(res)]
-        return cls._make_response(res, q, page, page_size, obs)
+        return cls._make_response(res, q, page, page_size, sort, obs)
+
+    @classmethod
+    def search_applications(cls, account, q, page, page_size, sort=None):
+        search_subs = app.config.get("DISCOVERY_APPLICATION_SEARCH_SUBS", {})
+        sort_subs = app.config.get("DISCOVERY_APPLICATION_SORT_SUBS", {})
+        query, page, page_size = cls._make_application_query(account, q, page, page_size, sort, search_subs, sort_subs)
+
+        # execute the query against the articles
+        res = models.Suggestion.query(q=query.query(), consistent_order=False)
+
+        # check to see if there was a search error
+        if res.get("error") is not None:
+            app.logger.error("Error executing discovery query search: {x}".format(x=res.get("error")))
```

Comment:
Wise to include a magic id for debugging support reqs and api feedback? Like this: https://github.com/DOAJ/doaj/blob/develop/portality/view/account.py#L178 ?